tests/testthat/test-misc.R

```diff
@@ -1,5 +1,8 @@
 context("test-misc.R")
 
+swe_set_ephe_path('')
+swe_set_topo(0,50,10)
```

Comment:
From my POV this is unnecessary. In all the tests that request topocentric positions, `swe_set_topo` is already used.